examples/image/draw_over/main.neva

```diff
@@ -0,0 +1,33 @@
+import {
+  io
+  image
+}
+
+component Main(start) (stop) {
+  nodes {
+    // bgReader io.ReadAll,
```

Comment:
remove comments or example itself

Reply:
Decided to remove the example as `Draw` is going away